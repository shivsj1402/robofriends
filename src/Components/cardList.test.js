import React from "react";
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import CardList from "./cardList";

const mockRobots=[{
    id: 1,
    name: 'Leanne Graham',
    username: 'Bret',
    email: 'Sincere@april.biz'
}]

it('cardList component render', ()=>{
    expect(toJson(shallow(<CardList robots={mockRobots}/>))).toMatchSnapshot();
})
