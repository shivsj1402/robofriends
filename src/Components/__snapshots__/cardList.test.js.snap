// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cardList component render 1`] = `
<div>
  <Card
    email="Sincere@april.biz"
    id={1}
    key="0"
    name="Leanne Graham"
  />
</div>
`;
